mithril-stm/src/stm.rs

```diff
@@ -902,13 +902,14 @@ impl CoreVerifier {
     ///     * Calculate the total stake of the eligible signers,
     ///     * Sort the eligible signers.
     pub fn setup(public_signers: &[(VerificationKey, Stake)]) -> Self {
-        let total_stake: Stake = 0;
+        let mut total_stake: Stake = 3;
```

Comment:
Do we want to start at `3` for a specific reason?
```suggestion
        let mut total_stake: Stake = 0;
```

Reply:
Ahahah! Leftover from debugging the code. :)